dcc-submission-server/src/main/java/org/icgc/dcc/submission/server/sftp/SftpSessions.java

```diff
@@ -38,4 +45,16 @@ public static Authentication getAuthentication(Session session) {
     return session.getAttribute(SESSION_KEY);
   }
 
+  public static void setFileTransfer(@NonNull Session session, @NonNull FileTransfer fileTransfer) {
+    session.setAttribute(FILE_TRANSFER_SESSION_ATTRIBUTE, fileTransfer);
+  }
+
+  public static void unsetFileTransfer(@NonNull Session session) {
+    session.setAttribute(FILE_TRANSFER_SESSION_ATTRIBUTE, NO_FILE_TRANSFER);
```

Comment:
I think the contract would be confusing in such a case. A `Session` can only have 1 active transfer using the current interface.